### DISCUSSION TITLE
Gh- pages branch not visibele in Sources

### DISCUSSION THREAD

**Discussion by ssat4805** [community/community#22078] [Other Feature Feedback, Questions, & Ideas]
<p>I created a web site using create react app. I deployed the react app using the react <a href="https://create-react-app.dev/docs/deployment/" rel="nofollow noopener">official documentation</a> . but When I am going to change gh-pages branch in the source, the source is not showing gh-pages branch. but gh-pages branch is already there in the repo. When I ran the command npm run deployed it run successfully. here is my repo link <a href="https://github.com/sabesansathananthan/sabesansathananthan.github.io" rel="nofollow noopener">https://github.com/sabesansathananthan/sabesansathananthan.github.io</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/a7943a1fdd6c6b7a2e40e4c55c6639329cd73b74.png" data-download-href="/uploads/short-url/nUtjnrWQd4OJcZJSgLTb0rg2tko.png?dl=1" title="Screenshot (40).png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/36799589/181068797-c20c98ec-c11b-486b-a5a2-efdab8be95ac.png" alt="Screenshot (40).png" data-base62-sha1="nUtjnrWQd4OJcZJSgLTb0rg2tko" width="690" height="316" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/a7943a1fdd6c6b7a2e40e4c55c6639329cd73b74_2_690x316.png, https://aws1.discourse-cdn.com/github/optimized/1X/a7943a1fdd6c6b7a2e40e4c55c6639329cd73b74_2_1035x474.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/a7943a1fdd6c6b7a2e40e4c55c6639329cd73b74_2_1380x632.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/a7943a1fdd6c6b7a2e40e4c55c6639329cd73b74_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot (40).png</span><span class="informations">1882×864 64.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/664d88de263ba91d59ec495637d9633103993ba7.jpeg" data-download-href="/uploads/short-url/eB0OMWNfsCHgKzOjTvbkuZWJp0H.jpg?dl=1" title="Capture.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/36799589/181068799-1bdde5b6-d936-4db7-92ee-0a1769dc47c7.jpg" alt="Capture.JPG" data-base62-sha1="eB0OMWNfsCHgKzOjTvbkuZWJp0H" width="690" height="343" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/664d88de263ba91d59ec495637d9633103993ba7_2_690x343.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/664d88de263ba91d59ec495637d9633103993ba7_2_1035x514.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/664d88de263ba91d59ec495637d9633103993ba7_2_1380x686.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/664d88de263ba91d59ec495637d9633103993ba7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture.JPG</span><span class="informations">1898×944 89.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/sabesansathananthan">@sabesansathananthan</a>,</p>
<p>Thanks for being here! I’m able to see your paged deployed at<a href="https://github.com/sabesansathananthan/sabesansathananthan.github.io" rel="nofollow noopener">sabesansathananthan.github.io</a>, if you are still having issues please let us know.</p>

**Comment by ssat4805** [comment]
<p>Thank you. It was fixed.</p>
